### DISCUSSION TITLE
perguntas

### DISCUSSION THREAD

**Discussion by MichelBNasc** [MichelBNasc/StrONG-first-version#1] [Q&A]
Tenho duvida em entender esse código 

const container = document.createElement("div");

container.id = "container";

document.body.appendChild(container);

const border = document.createElement("div");

border.classList.add("border");

div.appendChild(border);

**Comment by AyrtonSilveira** [comment]
Opa meu amigo, é simples neste código voce esta criando um elemento HTML via javascript, espero ter ajudado e bom estudo!!!
